### PR TITLE
Jemalloc

### PR DESCRIPTION
**Pull request by generall**
Issue: https://github.com/qdrant/qdrant/issues/948

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -25,6 +25,8 @@ use storage::content_manager::consensus::persistent::Persistent;
 use storage::content_manager::consensus_state::{ConsensusState, ConsensusStateRef};
 use storage::content_manager::toc::TableOfContent;
 use storage::dispatcher::Dispatcher;
+#[cfg(not(target_env = "msvc"))]
```
Comment (IvanPleshkov) [inline src/main.rs:28]: Do we support windows and msvc compiler?

Comment (generall) [reply src/main.rs:28]: idk, that is from the jemalloc docs

Comment (generall) [reply src/main.rs:28]: it is better to keep this switch, I guess

Comment (IvanPleshkov) [reply src/main.rs:28]: If we don't support Windows, we can just remove this checks

Comment (generall) [reply src/main.rs:28]: what the point? If we remove and then enable the support in future, we will never going to find this place again.

Comment (IvanPleshkov) [reply src/main.rs:28]: Okay, let's keep it as is. but in my opinion, we don't need any mention of windows os

**Review by IvanPleshkov — APPROVED** [review]

**Review by agourlay — APPROVED** [review]

**Pull request merged**